mass_mailing_custom_unsubscribe_event/demo/assets.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Tecnativa - João Marques
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<template id="assets_frontend_demo"
+          inherit_id="web.assets_frontend">
+    <xpath expr=".">
+        <script type="text/javascript"
+                src="/mass_mailing_custom_unsubscribe_event/static/src/js/tour.js"/>
+    </xpath>
+</template>
+
+<!--This is set here to make tours work-->
+<template id="mass_mailing_custom_unsubscribe.layout"
+          inherit_id="mass_mailing.layout">
+    <xpath expr="//t[@t-set='head']" position="inside">
+        <t t-call-assets="web_editor.assets_wysiwyg"/>
+        <t t-call-assets="web.assets_frontend"/>
+    </xpath>
+</template>
```

Comment:
You have to use `t-call-assets="web.conditional_assets_tests"` instead.

See https://github.com/odoo/odoo/blob/c92f058571b0656451f24e24154dd1bdb3bdb48a/addons/web/views/webclient_templates.xml#L410-L412

I guess that should go directly into `mass_mailing_custom_unsubscribe`.